packages/typespec-python/src/emitter.ts

```diff
@@ -67,9 +67,23 @@ interface HttpServerParameter {
 const defaultOptions = {
     "package-version": "1.0.0b1",
     "generate-packaging-files": true,
-    "unbranded": false,
 };
 
+// case insensitive, known values are "azure" and others
+let flavor: string;
```

Comment:
is it possible to use context to store the flavor option?

Reply:
context has stored it while it is necessary to update the "flavor" with special logic and original value before using it. And it is not good solution to change the original value in context, so I adopt global var.

Reply:
we should definitely use context, we should never have global variables in our emitter, it leads to issues when running emitters alongside each other